improve Importer documentation (#217)
tidalmigrations/tidalmigrations.github.io: pages/user guides/importing_to_excel.md

You should have one sheet or file per type of record.

There are several default columns values included for each record type, 
however if you have additional information on your inventory that would like to include, you can create custom fields for a record (Check step 3). 
Remember to add those fields before importing the data.

## How to import your data

1. Select the data type you wish to import.

    {% include image.html file="step0.PNG" %}

2. Select the sheet in your file that you want to import data from.

    {% include image.html file="step1.PNG" %}

3.  Map the columns of your sheet to the fields from the drop down. If you have a column that is not present as a field and you want to import it, you can add that field.

    {% include image.html file="step2.PNG" %}

4. Click submit to import the data.

{% include note.html content="If you are importing applications, the _name_ field is required for each application in order to successfully import. For any of the other record types you are importing, there are no fields that are required in order for you to import them." %}

> {% include tip.html content="The Tags option can be used to generate multiple tags and associate them with the record. You can use this track any type of categorical data. Additionally you can search and filter for records based on tags. Multiple tags can be associated based on one column of data, they are parsed and split on any commas (,)" %}

Reviewer: this info:
>Multiple tags can be associated based on one column of data, they are parsed and split on any commas (,)

is probably useful to know all the different attributes that it is true for. ex URLs

Author: Do you think we should investigate ☝️ now before merging this PR. or something for us to do in the future?

Reviewer: i can't see any reason it is _needed_ in order to make any of the other changes already done here.

### Importing Virtualization Clusters

Default fields:
- Hypervisor Technology
- Servers

> There is no required field when importing Virtualization clusters. 

Reviewer: ```suggestion
There are no required fields when importing Virtualization clusters. 
```

> * _Hypervisor Technology_, this field is used to list the technologies you have in your inventory. 

Reviewer: ```suggestion
* _Hypervisor Technology_, this specifies the technology that is used to run the hypervisor. Ex. VMware ESXi, Hyper-V. 
```

> * _Servers_, this field is used to create associations as you import the rest of your inventory, in this instance, _Servers_ refers to the server (host name) hosting the hypervisor.

Reviewer: >this field is used to create associations as you import the rest of your inventory

I don't follow what this means.

> _Servers_ refers to the server (host name) hosting the hypervisor.

This is definitely accurate but i think not clear and maybe ambiguous, regarding 'hosting the hypervisor'. as there are other servers that host the control plane of the hypervisor. while these physical vms in terms of usage, host the virtual servers that apart of the cluster, and they do technically also host part of the hypervisor control plane to do that.

> A list of physical servers that are used to run all the virtual machines within the Cluster. The servers for each Cluster should be provided as a list of hostnames that are comma (,) separated.

Reviewer: >this field is used to create associations as you import the rest of your inventory

It took me some time but I get what this first part means now. The part that i was struggling with was _the rest of your inventory. I didn't follow what that referred to.

>As you import other records, (ex. Applications, Databases) this field is used to find the correct Server and create those dependencies.

That's just an example of what, in my case, would have avoided me not knowing at first.

Author: Nice, your last suggestion reads a lot better and reduces the extra complexity

    {% include note.html content="Make sure your server inventory (when importing virtual servers) contains a field `Cluster` that points back to the Virtualization Cluster it is part of." %}

### Importing Servers

> There is no required field when importing Servers. 

Reviewer: ```suggestion
There are no required fields when importing Servers. 
```

Default fields:
- Host Name (*)
- FQDN
- Assigned ID
- Zone
- Role
- RAM Allocated (GB)
- RAM Used (GB)
- CPU Count
- CPU Name
- Storage Allocated (GB)
- Storage Used (GB)
- Description
- IP Addresses -> Address (*)
- Database Instances -> Name (*)
- Environment -> Name (*)
- Cluster -> Host Name (*)
- Tags
- Operating System Version
- Operating System

**(*)** Associative fields.
* _Host Name_, This field is used to identify the server and to create association with Databases, Applications, and Virtualization Clusters.
* _Cluster -> Host Name_, If a server is a Virtual Server, this field should match the value from the _Servers_ field in the Virtualization Cluster inventory. The **cluster host name** field is be used create the necessary association with its Virtual Cluster and its physical host.
* _Database Instances -> Name_, If a server is hosting a Database (or multiple), this field should contain the **name**(s) of such database(s).
* _Environment -> Name_, This field is used to represent the server's environment.

### Importing Databases

> There is no required field when importing Databases.

Reviewer: ```suggestion
There are required fields when importing Databases.
```

Default fields:
- Name (*)
- Description
- Database Size (MB)
- Database Path
- Server -> Host Name (*)
- Environment -> Name (*)
- Tags

**(*)** Associative fields.

> * _Name_, This field is used to identify and connect a Database to an Application and a Server.

Reviewer: i do understand why _name_ is here, its used to associated the records. Maybe its just me in knowing both the data modelling and external usage, that I find it being here conflicting. When I read it, part of me sees it saying that when importing database records, this field is actively used to identify and connect the database to other records, but it doesn't do that. And internally I know, its not an 'associatian', its a plain old attribute. but maybe calling it associative is best, cause that's is use, except only when importing the other records? I guess really my confusion is this reads the exact same as the one for environment, but they do behave differently, and I can't from this doc alone discern that.

Author: Yeah, I see what you mean, @prees1. Maybe one of the issues is the word "connect". 

What about something like this:
```
_Name_, This field is used to identify your Database and it is helpful when associating a Database with an Application and a Server.
```

Reviewer: I get stuck on 'helpful' there, is that helpful for me in someway, if so how? or helpful to someone else, who? or to the software?

I'd take what you have and say why it is helpful _to them_ very clearly:

>_Name_, When you are importing other records, Applications and Servers, and you provide a set of dependent databases with them the _Name_ field for Database Instances is used to identify those databases and associate the databases as dependencies for those other records. This is helpful when you have some existing dependencies, between Databases and Servers or Applications, in your Excel files and want to capture and leverage them.

Author: Nice. Thanks a lot 👍
